tests/test_api.py

```diff
@@ -22,16 +22,21 @@
 
 class ApiTest(unittest.TestCase):
     @my_vcr.use_cassette("sc-login.yaml")
+    @my_vcr.use_cassette("sc-resolve-public-client-id.yaml")
     def setUp(self):
         config = Extension().get_config_schema()
-        config["auth_token"] = "1-35204-61921957-55796ebef403996"
+        config["auth_token"] = "3-35204-970067440-lVY4FovkEcKrEGw"
```

Comment:
Old auth token does not work anymore. As a result, the returned track uris in the test functions test_resolves_stream_track{_invalid_id} are different aswell.